src/v2/Apps/Consign/Components/GetPriceEstimate/ConsignArtistAutosuggest.tsx

```diff
@@ -12,18 +23,63 @@ export const ConsignArtistAutosuggest: React.FC = () => {
     suggestions,
   } = usePriceEstimateContext()
 
+  const trackFocusedOnSearchInput = () => {
+    tracking.trackEvent(
+      focusedOnSearchInput({
+        context_module: ContextModule.priceEstimate,
+        context_owner_type: OwnerType.consign,
+      })
+    )
+  }
+
+  const trackSelectedItemFromSearch = (
+    suggestion: ConsignSearchSuggestion["node"]
+  ) => {
+    tracking.trackEvent(
+      selectedItemFromSearch({
+        context_module: ContextModule.priceEstimate,
+        context_owner_type: OwnerType.consign,
+        owner_id: suggestion.internalID,
+        owner_slug: suggestion.slug,
+        owner_type: OwnerType.artist,
+        query: searchQuery,
+      })
+    )
+  }
+
+  const trackSearchedWithNoResults = () => {
+    tracking.trackEvent(
+      searchedWithNoResults({
+        context_module: ContextModule.priceEstimate,
+        context_owner_type: OwnerType.consign,
+        query: searchQuery,
+      })
+    )
+  }
+
+  useEffect(() => {
+    if (suggestions.length === 0) {
```

Comment:
Would this run every time a user changes the query? I wonder if we'd want some kind of `debounce` here so that we only capture the final query and not users typing out something new

Reply:
Ah good suggestion, will do